### ISSUE TITLE
Full support for Configuration and Policy Groups

### ISSUE THREAD

**Issue by lap-wingmen** [CiscoDevNet/terraform-provider-sdwan#249]
Any timeline for full support of Configuration and Policy Groups?

**Comment by danischm** [comment]
No.. we are actively working on it, but hard to say atm when it will be complete.

**Comment by lap-wingmen** [comment]
Thanks for your quick response. We are looking forward to it :-)

**Comment by lap-wingmen** [comment]
We have just found out that Cisco is prioritizing configuration groups for future and new installations. Therefore, it no longer makes sense for us to focus on device and feature templates. Can you please provide us with a more precise estimate for completion of configuration groups at the very least?

**Comment by seconroy** [comment]
As of release 0.7.0 both Configuration Groups and Policy Groups are now supported. We are aware and working resolve an issue related to importing existing Configuration/Policy Groups and aiming to include that fix in the next release
